### DISCUSSION TITLE
OctoPod Apple TV

### DISCUSSION THREAD

**Discussion by braincrush** [gdombiak/OctoPod#659] [General]
Hi, 

First of all amazing suite of apps!
I have a little problem though, Apple TV app give's me "Retrieving printers information" OctoPod works on my iPad and my iPhone on those same networks.
Is there anything I can do to make it work please?

Thanks
Seb

**Comment by gdombiak** [comment]
Hi Seb,

Make sure that OctoPod is allowed to use iCloud. To confirm this, in your phone go to Settings -> iCloud and click on Show all apps using iCloud. Verify that OctoPod is enabled.

Then open OctoPod from your iPhone or iPad and go to Settings -> Devices and click on "Reset iCloud data from printers". This will refresh printers information stored in your iCloud account.

Finally, close and reopen the Apple TV app. It will read the fresh data from your iCloud account. Make sure that AppleTV also has access to your iCloud of course. :)

HTH,
Gaston

**Reply by braincrush** [reply]
Thank you Gaston,

It's working fine now!

thank you
Seb
